File: Models/Opros.cs
using HackTonTemplate.Maps;

namespace HackTonTemplate.Models
{
    public class Opros
    {
        public virtual long Id { get; set; }
        public virtual User User { get; set; }
        public virtual DateTime Date { get; set; }
        public virtual bool HaveCommercialProject { get; set; }
    }
}

File: ClientApp/src/models/Opros.ts
import User from "./User";

export interface IOpros {
    id: number,
    user: User,
    date: Date,
    haveCommercialProject: boolean
}

export default class Opros {
    constructor(options: IOpros | null) {
        if(options == null)
            return;

        this.id = options.id;
        this.date = new Date(options.date);
        this.haveCommercialProject = options.haveCommercialProject;
        this.user = new User(options.user);
    }

    id = 0;
    date = new Date();
    haveCommercialProject = false;
    user = new User(null);
}